main.go

```diff
@@ -26,6 +26,9 @@ const (
 )
 
 var (
+	// envAccountsJWKSFile holds the name of the environment variable which
+	// holds the path to the JWKS file we need to use. Optional.
+	envAccountsJWKSFile = "ACCOUNTS_JWKS_FILE"
```

Comment:
Follow Up: We need to make sure that this gets follow up all the way through our documentation. This will need to be added to the `.env` file right?

Reply:
It defaults to `/accounts/conf/jwks.json`. So, if you don't set anything in your `.env`, it will look for the file there.
I'll add some explanation to the README.

Reply:
👍